tests/sqllogictests/suites/stage/options/case_sensitive_parquet.test

```diff
@@ -92,4 +92,17 @@ copy into t2 from @s1/t2 return_failed_only=true column_match_mode=case_sensitiv
 query
 select * from t2
 ----
-1 2
\ No newline at end of file
+1 2
+
+statement ok
+create or replace table t3 (name string, age int64, score double);
+
+statement ok
+copy into t3 from @data/parquet/uppercase_fields.parquet file_format = (type = 'PARQUET');
```

Comment:
@KKould 
the new test add can pass on main too.
what exactly is the bad case to fix for this pr.

Reply:
` copy /*+set_var(parquet_fast_read_bytes=1)*/ into .. `  fail on main branch